containers/dotnetcore-mssql/.devcontainer/mssql/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
Is this Dockerfile needed anymore? You could just reference the MSSQL image in docker-compose.yml to keep things simple.